app/src/main/java/emu/skyline/data/DataItem.kt

```diff
@@ -39,6 +39,11 @@ data class AppItem(private val meta : AppEntry) : DataItem() {
      */
     val uri get() = meta.uri
 
+    /**
+     * The string used as the sub-title, we currently use the author
+     */
```

Comment:
`The application version` or something like that.

Reply:
Oh yeah, i think i forget to change it after a copy and paste :) thks